src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -189,25 +189,19 @@
         <Grid
                       Grid.Row="2">
           <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
             <RowDefinition Height="*" />
             <RowDefinition Height="auto" />
           </Grid.RowDefinitions>
-          <!-- providers -->
-          <nuget:PackageManagerProvidersLabel
-                        Grid.Row="0"
-                        FormatString="{x:Static nuget:Resources.Label_UseInstead}"
-                        DataContext="{Binding Providers}" />
           <!-- Summary -->
           <TextBlock
-                        Grid.Row="1"
+                        Grid.Row="0"
```

Comment:
How does it look with deprecation messages and/or vulnerability information?

Reply:
Great question. Double-checked:

![image](https://user-images.githubusercontent.com/49205731/129778331-80edd232-3dbd-467c-a10e-05ce81e03657.png)
![image](https://user-images.githubusercontent.com/49205731/129778531-a231d8b1-245a-4c58-a9a5-f41532a748ad.png)

These look good to you?

Reply:
Looks great!